core/src/util/jobQueue.cpp

```diff
@@ -0,0 +1,40 @@
+#include "jobQueue.h"
+#include <mutex>
+
+namespace Tangram {
+
+std::unordered_map<JobQueue::ThreadId, std::vector<JobQueue::Job>> JobQueue::jobsForThreads;
+std::mutex jobMutex;
```

Comment:
I'm not sure I follow. This declaration creates static storage duration with no external visibility, which seems to be what we need here.